packages/block-library/src/video/edit.js

```diff
@@ -271,6 +290,24 @@ function VideoEdit( {
 						</div>
 					</MediaUploadCheck>
 				</PanelBody>
+				<ToolsPanel>
```

Comment:
I.e. instead of adding a toolspanel here, just add the DimensionsTool to the existing PanelBody right above.

Reply:
Initially, I attempted that; however, adding `DimensionsTool` within `PanelBody` doesn't display in the controls.🤔

Reply:
That's why I need to overrider top border with CSS.

Reply:
The `DimensionsTool` is intended for use within the context of a `ToolsPanel`. You can see the [aspect ratio](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/dimensions-tool/aspect-ratio-tool.js?plain=1#L81), [scale](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/dimensions-tool/scale-tool.js#L100), and [width/height controls](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/dimensions-tool/width-height-tool.js#L75) using the `ToolsPanelItem` in their respective files.

The `ToolsPanel` component manages the context and state for the panel determining when controls should be displayed or not. Attempting to use the `DimensionsTool` outside of a `ToolsPanel` means nothing is telling the `DimensionsTool` to render its controls.